screenpipe-app-tauri/app/page.tsx

```diff
@@ -66,17 +72,28 @@ export default function Home() {
           description: `switched to ${profile} profile, restarting screenpipe now`
         });
 
-        await invoke("kill_all_sreenpipes");
-    
-        await new Promise((resolve) => setTimeout(resolve, 1000));
-    
-        await invoke("spawn_screenpipe");
-    
+        await restartScreenpipe();
         await new Promise((resolve) => setTimeout(resolve, 1000));
         relaunch();
       })
     ]);
 
+    listen('shortcut-start-audio', async () => {
+      toast({
```

Comment:
this seems wrong

why are you not using the rest API to stop/start audio device?

should not interrupt the vision recording